async-nats/examples/jetstream_publish.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020-2022 The NATS Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::env;
+
+use async_nats::jetstream;
+
+#[tokio::main]
+async fn main() -> Result<(), async_nats::Error> {
+    // Use the NATS_URL env variable if defined, otherwise fallback to the default.
+    let nats_url = env::var("NATS_URL").unwrap_or_else(|_| "nats://localhost:4222".to_string());
+
+    // Create an unauthenticated connection to NATS.
+    let client = async_nats::connect(nats_url).await?;
+
+    // Access the JetStream Context for managing streams and consumers as well as for publishing and subscription convenience methods.
+    let jetstream = jetstream::new(client);
+
+    // Publish a few messages to the stream
+    for i in 0..10 {
+        jetstream
```

Comment:
Please at least show both ack and no ack interactions and explain the differences between them.
Otherwise, this example shows less than docs.

Reply:
Will do!